src/components/register/ErrorMessage/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const ErrorMessage = ({ message }) => {
+  if (message) {
+    return (
+      <div className="loginContainer__loginDialog__errorContainer">
+        {errorMessage}
+      </div>
+    );
+  } else return null;
```

Comment:
never do this, always add braces

and if (!message) {
return null
}

then you get to save indentation on the 2nd half

Reply:
PTAL